lib/env.js

```diff
@@ -1,6 +1,16 @@
 const envalid = require('envalid')
+const parseURL = require('url').parse
 const { str, url, num } = envalid
 
+if (process.env.GITHUB_HOST) {
+  try {
+    const parsed = parseURL(process.env.GITHUB_HOST)
+    process.env.GITHUB_URL = `${parsed.protocol}//${parsed.hostname}`
```

Comment:
`hostname` does not include port number, maybe better to use `parsed.host`? Or are port numbers a non-issue  here?